manufacturing_order_type/security/security.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <record id="manufacturing_order_type_multi_company_rule" model="ir.rule">
+        <field name="name">Manufacturing Order Type multi-company</field>
+        <field name="model_id" ref="manufacturing_order_type.model_manufacturing_order_type" />
+        <field name="domain_force">
+            ['|',('company_id','=',False),('company_id','in',company_ids)]
```

Comment:
```suggestion
            [('company_id','in',[False, company_ids])]
```
cc @pedrobaeza  Can we please update this on https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration as well ?

Reply:
Current guideline is:

![Selección_001](https://user-images.githubusercontent.com/7165771/83171051-c9c90100-a115-11ea-9e37-863425b495fb.png)

and your form is not correct, as you are putting a list inside another list (example: `[False, [1, 2, 3]]`)